docs/examples/clowdenv.yml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: cloud.redhat.com/v1alpha1
+# Custom Resource defined as part of the Clowder API
+kind: ClowdEnvironment
+metadata:
+  name: env-jumpstart
+spec:
+  targetNamespace: jumpstart
+  # Providers all your app to consume configuration 
+  # data automatically based on your request
+  providers:
+    # provides a k8s service on port 8000
```

Comment:
I'd opt to put some blank lines in here to make the separation easier on the eyes